### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by JoeChapman**
Updated the old docs with es6 
fixed typos and errors
moved docs to a 'docs' dir
linked from readme to docs/
add start-controller to exports

### CODE REVIEW

File: docs/confirm-controller.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+In config page template
+
+```html
+{{#tableSections}}
+  {{> partials-summary-table}} <!-- {{name}}, {{value}} and {{step}} are available in this scope -->
+{{/tableSections}}
+```
+
+## Mixins
```
Comment (joefitter) [inline docs/confirm-controller.md:44]: this isn't part of the confirm controller but rather a separate mixins module

**Comment by joefitter** [comment]
this LGTM apart from `mixins` isn't part of the confirm controller - should be a separate file perhaps? Also worth noting we should now remove forking docs/tests/logic from hof-controllers - I'll submit an issue

**Comment by joefitter** [comment]
Ah I already did - https://github.com/UKHomeOffice/hof-controllers/issues/46

**Comment by JoeChapman** [comment]
The mixins part I thought should be rmeoved from these docs altogether and put in the view docs on hof.
But I didn't do it here, because it was already part of these docs, not something I was adding. therefore another pr, once it's been added elsewhere

**Comment by JoeChapman** [comment]
I can remove the forking stuff, but where is it documented? I suppose it should be documented in the HOF docs. It is a feature you can use when using hof, where in the architecture it lives is not important

**Comment by joefitter** [comment]
True, it does relate to views, rethinking it I think it should perhaps be a middleware that is only executed on GET requests

**Comment by easternbloc** [comment]
@JoeChapman what's going on with this?

**Pull request closed without merging**